.github/workflows/container-healthchecks.yml

```diff
@@ -39,14 +39,18 @@ jobs:
           export -p | sed 's/declare -x //'
           COMPOSE_PROFILES="platform" ./gradlew :dockerComposeUp
 
-      - name: "Start containers: mocks and svc-*"
-        shell: bash
-        run: |
-          source scripts/setenv.sh
-          # Build and start Kafka mock required for svc-bie-kafka
-          ./gradlew -p mocks :mock-bie-kafka:docker
-          COMPOSE_PROFILES="kafka" ./gradlew -p mocks :dockerComposeUp
-          COMPOSE_PROFILES="platform,svc" ./gradlew :dockerComposeUp
+      # NOTE: svc-bie-kafka workflows are tied to inactive services and commented out to prevent unnecessary execution and updates.
+      # Documentation:
+      # - Disabling issue: https://github.com/department-of-veterans-affairs/abd-vro/issues/3802
+      # - BIE Docs: https://github.com/department-of-veterans-affairs/abd-vro/issues/3753
+      # - name: "Start containers: mocks and svc-*"
+      #   shell: bash
+      #   run: |
+      #     source scripts/setenv.sh
+      #     # Build and start Kafka mock required for svc-bie-kafka
+      #     ./gradlew -p mocks :mock-bie-kafka:docker
+      #     COMPOSE_PROFILES="kafka" ./gradlew -p mocks :dockerComposeUp
+      #     COMPOSE_PROFILES="platform,svc" ./gradlew :dockerComposeUp
```

Comment:
We should still use COMPOSE_PROFILES="platform,svc" for the next builds, it looks like ep-merge is [failing](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/12130970180/job/33822290526?pr=3814#step:10:41) its health check further down in this action.  I imagine it's because our other platform services aren't running.

Reply:
A yes, let's see if [this commit](https://github.com/department-of-veterans-affairs/abd-vro/pull/3814/commits/64c2f4d26db9788de4ee598985c22f6d234011cc) fixes the failure.